### DISCUSSION TITLE
Error: You don't have access to the domain [custom-domain] under [team-name]

### DISCUSSION THREAD

**Discussion by emmanuelhavet** [vercel/community#6290] [Help]
### Summary

Hello,

I would like to apply custom domain alias to my vercel deployment url from within my gitlab-ci.yml but I get the following error:

`Error: You don't have access to the domain [custom-domain-I-want-to-alias] under [team-name]`

### Example

This code is executed from within my gitlab-ci.yml

`
vercel alias set [deployment-url] [custom-domain] --token=<my_token> --scope=<team_token>
`

but this error is thrown:

`Error: You don't have access to the domain [custom-domain-I-want-to-alias] under [team-name]`

when I try the same command with another custom domain:

`Success! [another-custom-domain-I-don't-want-to-alias] now points to [deployment-url] `

Here a screenshot of what's I see on the vercel IHM:

![Screenshot from 2024-03-27 15-58-22](https://github.com/vercel/community/assets/165178755/086534a5-2291-4f25-b1d5-7fbcf50b419f)

note: The issue is even when adding the TXT record and completing verification. I just let it for let you check the error message.

```
Another Vercel account is using this domain. Set the following TXT record on _vercel.[custom.domain] to use [subdomain.custom.domain] in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.
```

- the domain I want to alias already have its A and CNAME pointed on correctly
- I'm team's owner

Is there a way to unlink this domain for all vercel account except mine ?

My problem look like theses two following issues, but i couldn't find solution :
[You don't have access to the domain](https://github.com/orgs/vercel/discussions/1826)
[Domain possibly owned by another account?](https://github.com/orgs/vercel/discussions/5053)

Thanks for help
Emmanuel

**Comment by amyegan** [comment]
Hey @emmanuel-brainsecurity. It looks like you need to set the TXT record to [verify domain access](https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access). That will let you alias the domain in your project. Don't forget that it can take some time for DNS record changes to propagate. We also have some [domain troubleshooting](https://vercel.com/docs/projects/domains/troubleshooting) tips in case you want to dig into the records yourself.

**Reply by emmanuelhavet** [reply]
Hello amyegan,

The issue is even when adding the TXT record and completing verification. I just let it in my post for let readers check the error message. The reason for what I need to set the TXT record seem to be the reason why I can't alias my domain. Because I don't need to set the TXT record for my others domains. There is only one domain I can't alias and this is the only domain for which I have to set the TXT record.

I read the doc about domain troubleshooting for a while but I couldn't found any solutions at my problem. So if someone have a lead to follow, i'm listening :)

**Reply by amyegan** [reply]
That particular domain may have previously been associated with another account. Not every domain requires this verification. You need to use the TXT record to verify that you own the domain.

**Reply by amyegan** [reply]
You can check global network propagation status by using a tool like https://www.whatsmydns.net/#TXT/_vercel.example.com, where you can replace the example domain with your `_vercel.{DomainIWantToAlias}` value.

**Reply by emmanuelhavet** [reply]
This is what i get when I follow the last instructions =>

![Screenshot from 2024-03-27 19-36-30](https://github.com/vercel/community/assets/165178755/44303c0c-dcc4-4c26-b25e-e423483d6cb8)

I can see my domain in the list of nameservers but I still get this error message when I try to alias this domain:
Error: You don't have access to the domain [domain.to.alias] under [team].

**Reply by amyegan** [reply]
Your account may recently have been updated to a new structure. In that case your domain may need to be moved from your account to your new hobby team. You can learn more about that in https://github.com/vercel/community/discussions/6298
